### DISCUSSION TITLE
Teclado nativo não funciona corretamente nos dispositivos mobile

### DISCUSSION THREAD

**Discussion by simaodeveloper** [frontendbr/forum#391] [Perguntas e Respostas]
Galera, estou com um problema chato, praticamente com o projeto finalizado me deparei com um problema, independente do dispositivo seja tablet ou smartphone, android ou ios, quando dou focus no input (simplesmente clico no input) para digitar, o teclado nativo abre e fecha logo em seguida, executando uma ação de blur no input também, alguém já passou por isso antes?

**Comment by simaodeveloper** [comment]
@vitorbritto @fdaciuk @LFeh @Mendrone @willianjusten @woliveiras

**Comment by fdaciuk** [comment]
Está usando alguma lib ou framework específico que possa estar adicionando esse comportamento?

**Comment by simaodeveloper** [comment]
Pior que não, no máximo jQuery e TweenMax

**Comment by leandroruel** [comment]
estranho, deve ser alguma coisa do js... vc está testando em algum simulador?
